### ISSUE TITLE
Will you make the pre-training code and training dataset public?

### ISSUE THREAD

**Issue by handsomelys** [invictus717/MiCo#2]
I admire and am interested in your work and would like to follow up on your work. Will you make the pre-training code and training dataset public?

**Comment by invictus717** [comment]
Yes! We promise all these data and code will be publicly available. We are hard-working cleaning up code and preparing documents for pretraining, fine-tuning, and instruction tuning with LLMs, while the only problem is such a huge engineering task. 
We plan to release a series of pretrained multimodal LLMs in these several days.

**Comment by handsomelys** [comment]
> Yes! We promise all these data and code will be publicly available. We are hard-working cleaning up code and preparing documents for pretraining, fine-tuning, and instruction tuning with LLMs, while the only problem is such a huge engineering task. We plan to release a series of pretrained multimodal LLMs in these several days.

This is so exciting! I’m really looking forward to your follow-up work!
